lib/protocon/plugins/driver_tcp.py

```diff
@@ -92,10 +96,16 @@ def open(self):
 			self._addrinfo = self._addrinfo._replace(sockaddr=self._addrinfo.sockaddr[:3] + (scope_id,))
 
 		tcp_sock = socket.socket(self._addrinfo.family, self._addrinfo.type)
+		context = ssl.create_default_context()
+		context.check_hostname = False
+		context.verify_mode = ssl.CERT_NONE
 		if self.settings['type'] == 'client':
-			tcp_sock.connect(self._addrinfo.sockaddr)
-			self._connection = tcp_sock
+			ssl_sock = context.wrap_socket(tcp_sock)
```

Comment:
It looks like this is going to *always* be SSL and removes support for the `tcp*` schemes.

Reply:
(Commenting from work account.)

Wow, I actually butchered this PR. Not even the example is right. Absolutely _do not_ merge this. 😅

Reply:
I think when I removed the fledgling server mode, I did not refactor to account for the change. My bad!

Reply:
Actually, I think I totally just goofed this code by being fixated on SSL. Fixed now.

Reply:
Example fixed.